### DISCUSSION TITLE
Potential data sources

### DISCUSSION THREAD

**Discussion by omri374** [omri374/address_detection#9] [Research]
1. Fake data
   - [Faker](https://faker.readthedocs.io/en/master/)
   - [Mimesis](https://mimesis.readthedocs.io/)
   - [FakeNameGenerator](https://www.fakenamegenerator.com/)

2. Real data
   - [Web scraping to extract contact information](https://medium.com/@rodrigonader/web-scraping-to-extract-contact-information-part-1-mailing-lists-854e8a8844d2)

3. Existing datasets
  - [Dataset used in the `address_extraction` package](https://github.com/bagrii/address_extraction/tree/master/dataset)

**Comment by fatemerhmi** [comment]
https://generatedata.com/
Generates street address, postal/zip, Region and country.
